app/helpers/application_helper.rb

```diff
@@ -1,4 +1,12 @@
 module ApplicationHelper
   include ::SufiaHelper
   include ContentBlockHelper
+
+  def nav_link(href)
```

Comment:
Good addition! Would like to see this extracted into Sufia, but that can happen later.